.rhcicd/pr_check_fp.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -exv
+
+# Clowder config
+export APP_NAME="policies"
+export COMPONENT_NAME="policies-ui-backend"
+export IMAGE="quay.io/cloudservices/iqe-tests:latest"
+export DEPLOY_TIMEOUT="600"
+
+# IQE plugin config
+export IQE_PLUGINS="floorist"
```

Comment:
Can we have this together with the main policies plugin and have it all in one PR check (file)?

Reply:
What gating properties are you concerned about?

Can you elaborate on what difficulties there were when running 2 plugins with one PR check?

Reply:
I recommend to extend the original PR check. If the IQE env variables cannot be combined, then we can run first (original) tests, then change the IQE env vars (that would set the floorist IQE plugin to run) and then run the `cji_smoke_test.sh` (or similar) second time.